packages/flutter/lib/src/rendering/image.dart

```diff
@@ -128,7 +128,6 @@ class RenderImage extends RenderBox {
   set colorBlendMode(BlendMode value) {
     if (value == _colorBlendMode)
       return;
-    _colorBlendMode;
```

Comment:
this one too, to catch that this fix is wrong because what it should do is set `_colorBlendMode = value` :-)

Reply:
done